spec/spec_helper.rb

```diff
@@ -26,7 +26,7 @@
   end
 
   config.after(:each) do
-    ActiveRecord::Base.connection.tables.each do |table|
+    ActiveRecord::Base.connection.select_values("SELECT tablename FROM pg_tables WHERE schemaname = 'public'").each do |table|
```

Comment:
I _think_ I did this because `ActiveRecord::Base.connection.tables` didn't include partitioned tables

Reply:
I added a comment. Appears to only be an issue on Rails 5.1 which is pretty interesting